get_http.go

```diff
@@ -115,6 +144,20 @@ func (g *HttpGetter) GetFile(dst string, u *url.URL) error {
 	return err
 }
 
+func (g *HttpGetter) parseUrl(u *url.URL) (basicAuthUser, basicAuthPass string) {
+	qUser := "http_basic_auth_user"
+	qPass := "http_basic_auth_pass"
```

Comment:
We need to document the format for users so that they can use this authentication.